masterlist.yaml

```diff
@@ -1791,7 +1791,7 @@ plugins:
   - name: 'MiniLoversSound.esm'
     req: [ *FNIS ]
   - name: 'Mjanas_Labyrinth.esp'
-    url: [ 'http://modsreloaded.com/mjanas-labyrinth' ]
+    url: [ 'https://modsreloaded.com/mjanas-labyrinth' ]
```

Comment:
This site doesn't appear to actually support HTTPS. The link's also dead, & the site doesn't appear to host mods anymore. However, the mod is available [here (NSFW)](https://www.loverslab.com/topic/9314-mjanas-labyrinth/), but it can also be removed as it doesn't appear to have many users & likely hasn't been updated in years.

Reply:
I'll agree, I'll remove the entry with the next round of updates.